doc_src/interactive.rst

```diff
@@ -713,8 +703,6 @@ Several commands are provided to interact with this directory history:
 - :doc:`prevd <cmds/prevd>` moves backward through the history. It is bound to :kbd:`alt-left` (``←``)
 - :doc:`nextd <cmds/nextd>` moves forward through the history. It is bound to :kbd:`alt-right` (``→``)
 
-.. _directory-stack:
-
```

Comment:
nice fixes, thanks. What's broken about these section IDs though?
When I remove this and re-run `ninja install`, I get

	/home/johannes/git/fish-shell/doc_src/cmds/cd.rst:53: WARNING: undefined label: 'directory-stack' [ref.ref]

and it's no longer a link